src/app/helptext/sharing/smb/smb.ts

```diff
@@ -224,6 +224,13 @@ export const helptext_sharing_smb = {
  <i>Allowed</i> denies all permissions by default except those that are manually defined.\
  <i>Denied</i> allows all permissions by default except those that are manually defined.'),
 
- formTitle: T('SMB')
+ formTitle: T('SMB'),
+
+ stripACLDialog: {
+    title: T('Warning'),
+    message: T('An ACL is detected on the selected path but <i>Enable ACL</i> is not selected for this share. \
+ ACLs must be stripped from the dataset prior to creation of an SMB share.'),
```

Comment:
Making sentence entirely active:
```suggestion
 ACLs must be stripped from the dataset prior to creating an SMB share.'),
```